torch/quantization/fuse_modules.py

```diff
@@ -45,12 +48,19 @@ def fuse_conv_bn_relu(conv, bn, relu):
     """
     assert(conv.training == bn.training == relu.training),\
         "Conv and BN both must be in the same mode (train or eval)."
-
     if conv.training:
-        return torch_fused.ConvBnReLU2d(conv, bn, relu)
+        assert not relu.inplace, 'We only support fusion of non-inplace ReLU.'
+        if isinstance(conv, torch.nn.Conv3d):
+            return torch_fused.ConvBnReLU3d(conv, bn, relu)
+        else:
+            return torch_fused.ConvBnReLU2d(conv, bn, relu)
```

Comment:
can you do the `if isinstance(conv, torch.nn.Conv3d):` separately before `if conv.training`?

Also please remove `assert not relu.inplace, 'We only support fusion of non-inplace ReLU.'`

Reply:
Yes, inplace ReLU fusion is now supported.